include/alpaka/acc/Traits.hpp

```diff
@@ -94,4 +101,27 @@ namespace alpaka
                 ::getAccName();
         }
     }
+
+    namespace queue
+    {
+        namespace traits
+        {
+            template<
+                typename TAcc,
+                typename TProperty>
+            struct Queue<
+                TAcc,
+                TProperty,
+                typename std::enable_if<
+                    concepts::ImplementsConcept<acc::ConceptAcc, TAcc>::value
```

Comment:
As a thought. Perhaps there is a sort of name collision between `concepts::ImplementsConcept` which checks if a concept is implemented (and this PR makes it public) and `concepts::Implements` which declares a concept is implemented. Maybe `concepts::ImplementsConcept` can be renamed to smth like `concepts::IsImplemented`?

Reply:
@psychocoderHPC do you plan to change it in this PR? It's just a name after all, not critical.

Reply:
I will not change the naming inside of this PR because it is out of the scope.